02-create.html

```diff
@@ -68,7 +68,7 @@ <h2 id="places-to-create-git-repositories" class="challenge panel panel-success"
 <span class="kw">mkdir</span> beta   <span class="co"># make a sub-directory alpha/beta</span>
 <span class="kw">cd</span> beta      <span class="co"># go into alpha/beta</span>
 <span class="kw">git</span> init     <span class="co"># make the beta sub-directory a Git repository</span></code></pre>
-<p>Why is it a bad idea to do this?</p>
```

Comment:
The version of this challenge you're changing is outdated. Sorry about that. We messed up at some point and lost the latest version of this challenge. See #96.